### DISCUSSION TITLE
part-ii/2015-80/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#257] [Question comments]
Part II 2015 Logic and Set Theory: Paper 1, Section II, I 

https://questions.tripos.org/part-ii/2015-80/

**Comment by ArthurConmy** [comment]
The final part is quite easy and is simply no, because there exists a valuation $v(p_i)=1 \forall i$ and hence $S$ cannot prove $p_1$.
